fairroot/parbase/FairRtdbRun.h

```diff
@@ -101,9 +101,7 @@ inline void FairRtdbRun::setRefRun(Int_t r)
     if (r == -1) {
         refRun = "";
     } else {
-        char name[255];
-        sprintf(name, "%i", r);
-        refRun = name;
+        refRun = Form("%i", r);
```

Comment:
What about [`TString::Form`](https://root.cern.ch/doc/master/classTString.html#aeb44d6183d8b1f1b7090dbd3c93f5e39)?

```suggestion
        refRun.Form("%i", r);
```

Also if we move to `fmt::format`, we should do it here as well?

Reply:
Changed to your suggestion. Does it make sense to include `fmt/core.h` in this header only for something
we can get with `TString`?